exercises/practice/knapsack/knapsack.rkt

```diff
@@ -0,0 +1,8 @@
+#lang racket
+
+(provide item maximum-value)
+
+(struct item (weight value) #:transparent)
```

Comment:
I don't think we need to mark `item` as transparent here since we're not relying on that implementation detail.

Reply:
OK, I thought that it might make it easier for students to play with it that way, but I will take it out.